### PR TITLE
Couchbase: export pillowfight pods log info to google sheet

### PR DESCRIPTION
**Pull request by shreekarSS**

### CODE REVIEW

**Review by Avilir — APPROVED** [review]
LGTM

File: tests/conftest.py [hunk 1]
```diff
@@ -2436,6 +2436,9 @@ def factory(replicas=3, run_in_bg=False, skip_analyze=False):
         couchbase.run_workload(replicas=replicas, run_in_bg=run_in_bg)
         # Run sanity check on data logs
         couchbase.analyze_run(skip_analyze=skip_analyze)
+        # Export data to the google sheet
+        couchbase.export_pfoutput_to_googlesheet(sheet_name='E2E Workloads', sheet_index=2)
```
Comment (PrasadDesala) [inline tests/conftest.py:2440]: Can we exclude importing the data to gsheet from this fixture?
This fixture can be called for any test(including failure tests) and IMO, there is no need to capture this data for failure tests.

File: tests/e2e/workloads/couchbase/test_couchbase.py [hunk 2]
```diff
@@ -2,33 +2,18 @@
...
-    def test_cb_workload_simple(self, couchbase):
+    def test_cb_workload_simple(self, couchbase_factory_fixture):
         """
         Testing basic couchbase workload
         """
-        couchbase.setup_cb()
-        couchbase.create_couchbase_worker(replicas=5)
-        couchbase.run_workload(replicas=5)
-        couchbase.analyze_run()
+        couchbase_factory_fixture(replicas=3)
```
Comment (PrasadDesala) [inline tests/e2e/workloads/couchbase/test_couchbase.py:19]: Please export data to the google sheet in this basic CB test.

**Review by vasukulkarni — DISMISSED** [review]

File: tests/conftest.py (hunk 1 above)
Comment (vasukulkarni) [reply tests/conftest.py:2440]: @PrasadDesala what index sheet is that?

Comment (vasukulkarni) [reply tests/conftest.py:2440]: Ignore I got it, If we can also post, pass/fail probably it would be easier to delete those failed runs.

Comment (shreekarSS) [reply tests/conftest.py:2440]: if the test is failed, I don't think so code will reach till export to the sheet,
a teardown would happen immediately

File: tests/e2e/workloads/couchbase/test_couchbase.py (hunk 2 above)
Comment (shreekarSS) [reply tests/e2e/workloads/couchbase/test_couchbase.py:19]: couchbase_factory_fixture() is handling export data as well

File: tests/conftest.py (hunk 1 above)
Comment (PrasadDesala) [reply tests/conftest.py:2440]: @vasukulkarni the failure tests I am referring to here are tier-4 tests.

File: tests/e2e/workloads/couchbase/test_couchbase.py (hunk 2 above)
Comment (PrasadDesala) [reply tests/e2e/workloads/couchbase/test_couchbase.py:19]: Yes, this was my continuation comment to above :)
Remove the gsheet export from the fixture and handle it in this test case.

**Review by wusui — APPROVED** [review]
Looks good.

**Review by PrasadDesala — APPROVED** [review]

**Review by vasukulkarni — APPROVED** [review]

**Pull request merged**